pkg/reconciler/v1alpha1/taskrun/resources/image_exporter.go

```diff
@@ -75,10 +75,8 @@ func AddOutputImageDigestExporter(
 				return xerrors.Errorf("Failed to format image resource data for output image exporter: %w", err)
 			}
 
-			for _, s := range taskSpec.Steps {
-				augmentedSteps = append(augmentedSteps, s)
-				augmentedSteps = append(augmentedSteps, imageDigestExporterContainer(s.Name, imagesJSON))
-			}
+			augmentedSteps = append(augmentedSteps, taskSpec.Steps...)
+			augmentedSteps = append(augmentedSteps, imageDigestExporterContainer(imagesJSON))
```

Comment:
at this point we probably don't even need `augmentedSteps` ? could use `taskSpec.Steps` directly